components/common/CustomHead/index.js

```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head';
+import Proptypes from 'prop-types';
+
+const CustomHead = ({ title, description, url, locale }) => {
+  // TODO: favicon 제작하여 추가하기
+  // html <head>에 들어갈 정보: https://github.com/joshbuchea/HEAD
+  const logoUrl =
+    'https://github.com/bear-bear-bear/mogakco/blob/master/public/assets/svg/logo.svg';
+
+  return (
+    <Head>
+      <meta charSet="UTF-8" />
+      <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <title>{title}</title>
+      <meta name="description" content={description} />
+
+      <meta property="og:site_name" content="Mogakco" />
+      <meta property="og:type" content="website" />
+      <meta property="og:title" content={title} />
+      <meta property="og:description" content={description} />
+      <meta property="og:url" content={url} />
+      <meta property="og:locale" content={locale} />
+      <meta property="og:image" content={logoUrl} />
+
+      <meta name="twitter:card" content="summary" />
+      <meta name="twitter:site" content="@mogakco" />
+      <meta name="twitter:title" content={title} />
+      <meta name="twitter:description" content={description} />
+      <meta name="twitter:image:src" content={logoUrl} />
+    </Head>
+  );
+};
+
+CustomHead.propTypes = {
```

Comment:
PropTypes 는 파일의 최상단에 배치하는게 좋습니다.
그래야 각 props 에 대한 정보를 확인하기 더 가독성이 좋습니다.
```suggestion
CustomHead.propTypes = {
```

Reply:
조언 너무 달달해요 하

Reply:
감사합니다. 말씀주신대로 작업하겠습니다

Reply:
조언 더해주세요 하아하아

Reply:
![image](https://user-images.githubusercontent.com/57123802/114543434-2bde1680-9c94-11eb-83c9-44df4cdf8652.png)

말씀해주신대로 작업시 TDZ 발생합니다 🙄